vertx-core/src/test/java/io/vertx/tests/file/FileResolverTestBase.java

```diff
@@ -486,4 +488,16 @@ public void testGetTheCacheDirWithoutHacks() {
       }
     }
   }
+
+  @Test
+  public void testGetTheExactCacheDirWithoutHacks() {
```

Comment:
this test should be moved to `FileCacheTest` instead, `FileResolverTestBase` tests the behaviour of resolver implementations